esp-hal/src/interrupt/mod.rs

```diff
@@ -86,22 +86,19 @@ pub mod software;
 
 #[cfg(feature = "rt")]
 #[unsafe(no_mangle)]
-extern "C" fn EspDefaultHandler(_interrupt: crate::peripherals::Interrupt) {
-    panic!("Unhandled interrupt: {:?}", _interrupt);
+extern "C" fn EspDefaultHandler() {
+    panic!("Unhandled interrupt");
```

Comment:
I suppose we're relying on the backtrace to figure out the unhandled interrupt now. I wonder if we can improve this (without the maybe/definitely UB signature casting we've been doing :D) 🤔.

Reply:
Agree, this is all unstable stuff right now.